DEVELOPERS.md

```diff
@@ -68,6 +68,18 @@
          `Console` tab to view printed logs and errors, the `Element` tab to
          inspect the HTML and CSS, the `Network` tab for network calls, etc).
 
+#### Debugging Add-in: Alternate way
+If the above method to debug the add-in is not working, one can try the following steps:
+   1. Delete the `C:\Users\{username}\.office-addin-dev-certs` folder.
+   2. Run `npx office-addin-dev-certs install`.
```

Comment:
Shouldn't this be `npx install office-addin-dev-certs`

Reply:
@algomaster99 
I am unsure about it as in [this comment](https://github.com/OfficeDev/office-js/issues/1852#issuecomment-840177792) it is stated `npx office-addin-dev-certs install`. I followed and it worked so I didn't try the other way round 😅